lib/bpf/bpf.go

```diff
@@ -165,20 +144,18 @@ func New(config *servicecfg.BPFConfig) (bpf BPF, err error) {
 	start := time.Now()
 	logger.DebugContext(closeContext, "Starting enhanced session recording")
 
-	// Compile and start BPF programs if they are enabled (buffer size given).
+	// Compile and start BPF programs (buffer size given).
 	s.exec, err = startExec(*config.CommandBufferSize)
 	if err != nil {
-		return nil, trace.Wrap(err)
+		return nil, trace.Wrap(err, "failed to load command execution hooks")
 	}
 	s.open, err = startOpen(*config.DiskBufferSize)
 	if err != nil {
-		return nil, trace.Wrap(err)
+		return nil, trace.Wrap(err, "failed to load file I/O hooks")
```

Comment:
Suggestion: Refer to these as disk instead of IO for consistency.
```suggestion
		return nil, trace.Wrap(err, "failed to load file disk hooks")
```